REGION_FORK_RECOMMENDATIONS.md

```diff
@@ -0,0 +1,216 @@
+1. Region specific changes will be limited in scope and never cause a merge conflict
+
+These recommendations exist because it is reasonable to assume that individual 
+Public Health Authorities may want to customize specific pieces of the Covid 
+Green codebase, but wish to do so in a way that will not cause explosive merge 
+conflicts. These recommendations work best when the set of region specific changes 
+are limited in scope, and unless explicitly region specific, all changes are 
+contributed back to [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api).
+
+As an example, [New York State's fork](https://github.com/project-vagabond/covid-green-backend-api) 
+contains NYS specific workflow differences which cannot be contributed back to Covid Green.
+
+## Branches
+
+The following items are assumed to be true for these recommendations to apply.
+
+1. `current` is the main development branch within Covid Green. 
+1. Covid Green's `current` branch may be merged into any fork's `current` branch without causing merge conflicts.
+1. `current` will never include region specific content
+
+The following branches within a region's fork, and rules associated with those 
+branches, will help each region stay up-to-date with Covid Green. Region specific 
+branches will never exist in the Covid Green project.
+
+For clarity, the table below assumes that the region is New York State (`nys`). 
+Other regions will potentially have their own region specific branch naming 
+convention. For example, Pennsylvania may use `penn`, New Jersey may use `njs`, 
+Connecticut may use `conn`. 
+
+<table>
+<tr>
+<th>Branch</th>
+<th>Description</th>
+<th>Reasoning</th>
+<th>Rules</th>
+</tr>
+<tr>
+<td>
+
+`current`
+
+</td>
+<td>
+
+Main code branch for the project.
+
+</td>
+<td>
+
+The purpose is to always be up-to-date with Covid Green and have a clean slate to pull into.
+
+</td>
+<td>
+
+1. The `current` branch will always track with Covid Green's `current` branch.
+
+1. Region specific forks will frequently pull `current` from Covid Green to ensure it is up-to-date.
+
+1. No PRs will be sent to or accepted into a fork's `current` branch.
+
+1. Pushing to a fork's `current` branch is not allowed, unless as a pull from Covid Green.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+`nys` (or `penn`, `njs`, etc...)
+
+</td>
+<td>
+
+Region specific branch tracking against `current`
+
+</td>
+<td>
+
+Allows regions to cleanly rebase off Covid Green without running into merge conflicts.
+
+</td>
+<td>
+
+1. This branch will always be based off `current`.
+
+1. This branch can contain files which don't exist in Covid Green's `current` branch.
+
+1. The only differences between `nys` and `current` should be files that do not exist in `current`.
+
+1. Changes in this branch are not allowed on files that exist in `current`.
+
+1. PRs can be accepted IFF they come from `nys-*` branches and do not contain changes to files in `current`.
```

Comment:
Is IFF a typo?

Reply:
No, it's not a typo. IFF is a mathematical conjunction for if-and-only-if. The full meaning of the sentence is 

> PRs can be accepted if and only if they come from ...

Reply:
My preference would be plain English in documentation

Reply:
Works for me. I'll make that change.